numpy/linalg/linalg.py

```diff
@@ -1615,17 +1615,20 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
         # reversing is cheap by design in numpy, so we do so to be consistent
         if compute_uv:
             s, u = eigh(a)
-            s = s[..., ::-1]
-            u = u[..., ::-1]
-            # singular values are unsigned, move the sign into v
-            vt = transpose(u * sign(s)[..., None, :]).conjugate()
+            sgn = sign(s)
             s = abs(s)
+            sidx = s.shape[-1] - 1 - argsort(s)
```

Comment:
Doesn't looks correct to me. Shouldn't this be the following?
```suggestion
            sidx = argsort(s)[::-1]
```

Reply:
Yes, that is what I was thinking of.